src/main/java/no/nav/arbeidsgiver/sykefravarsstatistikk/api/statistikk/sykefraværshistorikk/SykefraværshistorikkController.java

```diff
@@ -102,17 +118,48 @@
         SummertSykefraværshistorikk summertSykefraværshistorikkVirksomhet =
                 summertSykefraværService.hentSummertSykefraværshistorikk(
                         underenhet,
-                        new ÅrstallOgKvartal(2021, 2),
+                        SISTE_PUBLISERTE_ÅRSTALL_OG_KVARTAL,
                         antallKvartaler
                 );
 
         SummertSykefraværshistorikk summertSykefraværshistorikkBransjeEllerNæring =
                 summertSykefraværService.hentSummertSykefraværshistorikkForBransjeEllerNæring(
                         underenhet,
-                        new ÅrstallOgKvartal(2021, 2),
+                        SISTE_PUBLISERTE_ÅRSTALL_OG_KVARTAL,
                         antallKvartaler
                 );
 
         return Arrays.asList(summertSykefraværshistorikkVirksomhet, summertSykefraværshistorikkBransjeEllerNæring);
     }
+
+    @GetMapping(value = "/{orgnr}/sykefravarshistorikk/legemeldtsykefravarsprosent")
+    public LegemeldtSykefraværsprosent hentLegemeldtSykefraværsprosent(
+            @PathVariable("orgnr") String orgnrStr,
+            HttpServletRequest request
+    ) {
+        InnloggetBruker bruker = tilgangskontrollService.hentInnloggetBruker();
+
+        tilgangskontrollService.sjekkTilgangTilOrgnrOgLoggSikkerhetshendelse(
+                new Orgnr(orgnrStr),
+                bruker,
+                request.getMethod(),
+                "" + request.getRequestURL()
+        );
+        Underenhet underenhet = enhetsregisteretClient.hentInformasjonOmUnderenhet(new Orgnr(orgnrStr));
+
+        LegemeldtSykefraværsprosent legemeldtSykefraværsprosent =
+                summertLegemeldtSykefraværService.hentLegemeldtSykefraværsprosent(
+                        underenhet,
+                        SISTE_PUBLISERTE_ÅRSTALL_OG_KVARTAL
+                );
+
+        // TODO: null er ikke robust nok
```

Comment:
Hva med en "204 No Content" med forklarende feiltekster? Noe ala  
1. "Organisasjonen du søker på finnes i databasen, men ingen sykefraværsstatistikk er tilgjengelig for øyeblikket" og  
2. "Organisasjonen finnes ikke i databasen"

Reply:
Vi gikk for `204 - No Content` 
Men uten forklarende tekst da 204 returneres uten body